src/main/resources/init_db.sql

```diff
@@ -18,3 +18,29 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+CREATE TABLE taxi_service_db.cars (
```

Comment:
```suggestion
CREATE TABLE cars (
```
You shouldn't use database name here